docs/docs/api/objects/player-options.md

```diff
@@ -15,4 +15,5 @@ All parameters are optional. You also only need to specify the ones you want to
 | `waitForBuffer` | `boolean` | Indicates whether the player should automatically delay playback in order to minimize stalling. Defaults to `true`. @deprecated This option has been nominated for removal in a future version of RNTP. If you have this set to `true`, you can safely remove this from the options. If you are setting this to `false` and have a reason for that, please post a comment in the following discussion: https://github.com/doublesymmetry/react-native-track-player/pull/1695 and describe why you are doing so. | ✅ | ✅ |
 | `autoUpdateMetadata` | `boolean` | Indicates whether the player should automatically update now playing metadata data in control center / notification. Defaults to `true`. | ✅ | ✅ |
 | `autoHandleInterruptions` | `boolean` | Indicates whether the player should automatically handle audio interruptions. Defaults to `false`. | ✅ | ✅ |
+| `androidAudioUsageType` | `boolean` | (Android only) Specifies why the source is playing and controls routing, focus, and volume decisions. With `androidAudioUsageType` set to VoiceCommunication the audio will come from the earpiece speaker | ✅ | ❌ |
```

Comment:
@Egbert-Jan the type of this isn't a boolean, it should be the `AndroidAudioUsageType` enum. Then in the description you should use `AndroidAudioUsageType.VoiceCommunication`.